app/src/main/java/com/duckduckgo/app/browser/BrowserActivity.kt

```diff
@@ -140,6 +140,8 @@ open class BrowserActivity : DuckDuckGoActivity() {
 
     private var openMessageInNewTabJob: Job? = null
 
+    private var externalLaunch: Boolean = false
```

Comment:
Do we need to keep this here? Can we use `intent?.getBooleanExtra(LAUNCH_FROM_EXTERNAL_EXTRA, false) ?: false` when needed?